src/Symfony/Component/HttpKernel/EventListener/SaveSessionListener.php

```diff
@@ -44,19 +45,22 @@
  */
 class SaveSessionListener implements EventSubscriberInterface
 {
+    private $container;
```

Comment:
please add explicit typehint for this property, typehint for constructor parameter is not the same as typehint for property

Reply:
We actually removed such duplicate comments from the code base.

Reply:
:-1: on that, when one would do sth later with this code, he need to remember that perhaps he need to add that typehinting (eg, removing obtaining the parameter via constructor, changing visibility to public and so). I agree that for private it is no big deal, yet still.
And we do had already issues with field that was assumed to be type X, but in the end it was `null|X`, just not documented...

Reply:
I respect your opinion, but this is not the proper place to discuss this. Right now, this is just consistent with the code base.

Reply:
I treat it same as with params, as long as we cannot typehint it on lang level, we do it in docblock ;)
Would be great if we would be able to do it with any var, like here with the property, but we are not there yet ;)
Well, I'm bet I just saw too many fuckups so I'm extra defensive...